public/doc/openapi-3-v3.1.json

```diff
@@ -17,6 +17,40 @@
       "UserSecurity": []
     }
   ],
+  "tags": [
+    {
+      "description": "Endpoints related to application metadata",
+      "name": "app metadata"
+    },
+    {
+      "description": "Endpoints related to applications",
+      "name": "applications"
+    },
+    {
+      "description": "Endpoints related to application authentications",
+      "name": "application authentications"
+    },
+    {
+      "description": "Endpoints related to application types",
+      "name": "application types"
+    },
+    {
+      "description": "Endpoints related to authentication",
+      "name": "authentications"
+    },
+    {
+      "description": "Endpoints related to endpoints",
+      "name": "endpoints"
+    },
+    {
+      "description": "Endpoints related to sources",
+      "name": "sources"
+    },
+    {
+      "description": "Endpoints related to source types",
+      "name": "source types"
+    },
```

Comment:
I think this comma is blowing up the tests - if we remove it it should pass.

Reply:
Aaaarggh... thank you :+1: